src/utils.ts

```diff
@@ -425,21 +328,6 @@ export function getDaysPassed(from: Date, to: Date): number {
   return Math.floor((to.getTime() - from.getTime()) / (24 * 60 * 60 * 1000));
 }
 
-// Decode a Radicle Id (URN).
-export function decodeRadicleId(urn: string): Uint8Array {
-  const encoded = parseRadicleId(urn);
-  const multihash = multibase.decode(encoded);
-  const hash = multihashes.decode(multihash);
-
-  return hash.digest;
-}
-
-// Create a project hash from a hash and format.
-export function formatProjectHash(multihash: Uint8Array): string {
-  const decoded = multihashes.decode(multihash);
-  return ethers.utils.hexlify(decoded.digest).replace(/^0x/, "");
-}
```

Comment:
We're probably going to need all these methods again soon.

Reply:
We should use packages that aren't deprecated when we bring them back then.

```
npm WARN deprecated multibase@4.0.6: This module has been superseded by the multiformats module
```